### ISSUE TITLE
not unblocking 

### ISSUE THREAD

**Issue by szymixsiorek** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#265]
### What happened?

14:02:20 [INFO] Script injected (https://www.youtube.com/watch?v=J2XuYSx5jOo&ab_channel=melindalayla1369)
14:02:20 [INFO] playerResponse property set, contains sidebar: false
14:02:20 [INFO] Trying Player Unlock Method #2 (TV Embedded Player)
14:02:20 [INFO] Trying Player Unlock Method #4 (Account Proxy)
14:02:20 [ERROR] Proxy API Error; Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'https://youtube-proxy.zerody.one/getPlayer?videoId=J2XuYSx5jOo&reason=LOGIN_REQUIRED&clientName=WEB&clientVersion=2.20241025.01.00&signatureTimestamp=20019&startTimeSecs=0&hl=pl&isEmbed=0&isConfirmed=0&includeNext=1&client=js'.
    at self.XMLHttpRequest.E [as send] (data:text/javascript;base64,KGZ1bmN0aW9uKCl7ZnVuY3Rpb24gQihjLGEpe24uYXBwbHkodGhpcyxhcmd1bWVudHMpO3RyeXtpZih0LnRlc3QoYSkpe3ZhciBiPXRoaXMudXBsb2FkLl9faWRtX3JlcWlkX189InhycS0iK3UrIi0iKyArK3c7dGhpcy5hZGRFdmVudExpc3RlbmVyKCJsb2FkZW5kIixDLmJpbmQodGhpcyxiKSl9fWNhdGNoKGQpe319ZnVuY3Rpb24gRChjLGEpe3RyeXt0aGlzLnVwbG9hZC5fX2lkbV9yZXFpZF9fJiYiYWNjZXB0Ij09Yy50b0xvd2VyQ2FzZSgpJiYodGhpcy51cGxvYWQuX19pZG1fYWNjZXB0X189ITApfWNhdGNoKGIpe31rLmFwcGx5KHRoaXMsYXJndW1lbnRzKX1mdW5jdGlvbiBFKGMpe3RyeXt2YXIgYT10aGlzLnVwbG9hZCxiPWEuX19pZG1fcmVxaWRfXztpZihiKXt2YXIgZD1hLl9faWRtX2FjY2VwdF9fO2RlbGV0ZSBhLl9faWRtX2FjY2VwdF9fO2RlbGV0ZSBhLl9faWRtX3JlcWlkX187ZHx8ay5jYWxsKHRoaXMsImFjY2VwdCIsIiovKiIpO2suY2FsbCh0aGlzLCJhY2NlcHQiLAoiYXBwbGljYXRpb24vdm5kLnQxYy4iK2IpfX1jYXRjaChlKXt9cC5hcHBseSh0aGlzLGFyZ3VtZW50cyl9ZnVuY3Rpb24gQyhjKXt0cnl7aWYoYyl7dmFyIGE9dGhpcy5yZXNwb25zZVRleHQsYj12LmV4ZWMoYSk7cihbMTIyOTIxMjk3OSxjLGImJihiWzFdfHxhKXx8bnVsbF0sIi8iKX19Y2F0Y2goZCl7fX1mdW5jdGlvbiBGKGMsYSl7dmFyIGIsZCxlPXgoYyk7dHJ5e2lmKGI9dC50ZXN0KGU/Yy51cmw6Yykpe2U/ZD1jLmhlYWRlcnM6YT9kPWEuaGVhZGVyc3x8KGEuaGVhZGVycz1uZXcgSGVhZGVycyk6YT1hcmd1bWVudHNbMV09e2hlYWRlcnM6ZD1uZXcgSGVhZGVyc307dmFyIGw9ImZycS0iK3UrIi0iKyArK3c7ZC5oYXMoImFjY2VwdCIpfHxkLmFwcGVuZCgiYWNjZXB0IiwiKi8qIik7ZC5hcHBlbmQoImFjY2VwdCIsImFwcGxpY2F0aW9uL3ZuZC50MWMuIitsKX19Y2F0Y2goZil7fWQ9cS5hcHBseSh0aGlzLGFyZ3VtZW50cyk7dHJ5e2lmKGIpcmV0dXJuIGQudGhlbihHLmJpbmQodGhpcywKbCkpfWNhdGNoKGYpe31yZXR1cm4gZH1mdW5jdGlvbiBHKGMsYSl7dHJ5e2lmKGMpaWYoYS5vayl7dmFyIGI9YS50ZXh0LGQ9YS5qc29uO2EudGV4dD1mdW5jdGlvbigpe3JldHVybiBiLmNhbGwodGhpcykudGhlbih5LmJpbmQodGhpcyxjLCExKSl9O2EuanNvbj1mdW5jdGlvbigpe3JldHVybiBkLmNhbGwodGhpcykudGhlbih5LmJpbmQodGhpcyxjLCEwKSl9fWVsc2UgcihbMTIyOTIxMjk3OSxjXSwiLyIpfWNhdGNoKGUpe31yZXR1cm4gYX1mdW5jdGlvbiB5KGMsYSxiKXt0cnl7dmFyIGQ9YT9KU09OLnN0cmluZ2lmeShiKTpiLGU9di5leGVjKGQpO3IoWzEyMjkyMTI5NzksYyxlJiYoZVsxXXx8ZCl8fG51bGxdLCIvIil9Y2F0Y2gobCl7fXJldHVybiBifWZ1bmN0aW9uIHooYyxhKXt2YXIgYj1jO2E9YS5zcGxpdCgiLiIpO2lmKCIqIj09YVswXSl7YS5zaGlmdCgpO2Zvcih2YXIgZCBvZiBIKGMpKXRyeXtpZih4KGI9Y1tkXSkmJkkoYixhWzBdKSl7Zm9yKHZhciBlIG9mIGEpYj1iW2VdOwpyZXR1cm4gYn19Y2F0Y2gobCl7fX1lbHNlIHRyeXtmb3IoZSBvZiBhKWI9YltlXTtyZXR1cm4gYn1jYXRjaChsKXt9fWZ1bmN0aW9uIEEoYyxhKXtmb3IoO2MmJmEtLTtjPWMucGFyZW50RWxlbWVudCk7cmV0dXJuIGN9Y29uc3QgSj13aW5kb3cub3JpZ2lufHxkb2N1bWVudC5vcmlnaW58fGxvY2F0aW9uLm9yaWdpbix4PU9iamVjdC5pc0V4dGVuc2libGUsSz1BcnJheS5pc0FycmF5LGg9WE1MSHR0cFJlcXVlc3QucHJvdG90eXBlLHI9d2luZG93LnBvc3RNZXNzYWdlLmJpbmQod2luZG93KTt2YXIgdz0wLHU9MCx0LHYsbixwLGsscTtjb25zdCBJPU9iamVjdC5oYXNPd258fEZ1bmN0aW9uLmNhbGwuYmluZChPYmplY3QucHJvdG90eXBlLmhhc093blByb3BlcnR5KSxIPU9iamVjdC5nZXRPd25Qcm9wZXJ0eU5hbWVzLEw9LyAqXCQoWzw+PV0pICovO3dpbmRvdy5hZGRFdmVudExpc3RlbmVyKCJtZXNzYWdlIixmdW5jdGlvbihjKXt2YXIgYT1jLmRhdGE7aWYoSyhhKSYmYy5vcmlnaW49PQpKKXN3aXRjaChhWzBdKXtjYXNlIDEyMjkyMTI5Nzg6dT1hWzFdO3ZhciBiPWFbMl0sZD1hWzNdLGU9YVs0XTthPWFbNV07ZT8odD1SZWdFeHAoZSksdj1SZWdFeHAoYSkpOmI9ZD0hMTt0cnl7Yj8obnx8KG49aC5vcGVuLGgub3Blbj1CKSxwfHwocD1oLnNlbmQsaC5zZW5kPUUpLGt8fChrPWguc2V0UmVxdWVzdEhlYWRlcixoLnNldFJlcXVlc3RIZWFkZXI9RCkpOihuJiYoaC5vcGVuPW4sbj1udWxsKSxwJiYoaC5zZW5kPXAscD1udWxsKSxrJiYoaC5zZXRSZXF1ZXN0SGVhZGVyPWssaz1udWxsKSksZD9xfHwocT1mZXRjaCxmZXRjaD1GKTpxJiYoZmV0Y2g9cSxxPW51bGwpfWNhdGNoKE0pe31icmVhaztjYXNlIDEyMjkyMTI5ODA6Yz1hWzFdO3ZhciBsPWFbMl07YT1hWzNdO3ZhciBmO3RyeXtpZihsP2Y9ZG9jdW1lbnQucXVlcnlTZWxlY3RvcignW19faWRtX2lkX189IicrbCsnIl0nKTpmPXdpbmRvdyxmKWE6aWYoZD1hLnNwbGl0KEwpLDE8ZC5sZW5ndGgpe2ZvcihkLnNoaWZ0KCk7ZiYmCihlPWQuc2hpZnQoKSk7KXt2YXIgZz1kLnNoaWZ0KCkudHJpbSgpLG07Zm9yKG09MTsiPCI9PWdbZy5sZW5ndGgtbV07bSsrKTstLW0mJihnPWcuc2xpY2UoMCwtbSkudHJpbSgpKTtzd2l0Y2goZSl7Y2FzZSAiPCI6Zj1BKGc/Zi5jbG9zZXN0KGcpOmYsbSk7YnJlYWs7Y2FzZSAiPiI6Zj1BKGYucXVlcnlTZWxlY3RvcihnKSxtKTticmVhaztjYXNlICI9IjppZihnPXooZixnKSx2b2lkIDAhPT1nKXtiPWc7YnJlYWsgYX19fWI9dm9pZCAwfWVsc2UgYj16KGYsYSl9Y2F0Y2goTSl7fXIoWzEyMjkyMTI5ODEsYyxiXSwiLyIpfX0sITEpO3IoWzEyMjkyMTI5NzddLCIvIil9KSgpOwo=:2:40)

14:02:20 [ERROR] Video unlock failed; Player Unlock Failed, Proxy Error Message: Proxy Connection failed
    at unlockResponse$1 (<anonymous>:873:19)

14:02:21 [INFO] Trying Next Unlock Method #2 (Account Proxy)
14:02:21 [ERROR] Proxy API Error; Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'https://youtube-proxy.zerody.one/getNext?videoId=J2XuYSx5jOo&clientName=WEB&clientVersion=2.20241025.01.00&hl=pl&userInterfaceTheme=USER_INTERFACE_THEME_DARK&isEmbed=0&isConfirmed=0&client=js'.
    at self.XMLHttpRequest.E [as send] (data:text/javascript;base64,KGZ1bmN0aW9uKCl7ZnVuY3Rpb24gQihjLGEpe24uYXBwbHkodGhpcyxhcmd1bWVudHMpO3RyeXtpZih0LnRlc3QoYSkpe3ZhciBiPXRoaXMudXBsb2FkLl9faWRtX3JlcWlkX189InhycS0iK3UrIi0iKyArK3c7dGhpcy5hZGRFdmVudExpc3RlbmVyKCJsb2FkZW5kIixDLmJpbmQodGhpcyxiKSl9fWNhdGNoKGQpe319ZnVuY3Rpb24gRChjLGEpe3RyeXt0aGlzLnVwbG9hZC5fX2lkbV9yZXFpZF9fJiYiYWNjZXB0Ij09Yy50b0xvd2VyQ2FzZSgpJiYodGhpcy51cGxvYWQuX19pZG1fYWNjZXB0X189ITApfWNhdGNoKGIpe31rLmFwcGx5KHRoaXMsYXJndW1lbnRzKX1mdW5jdGlvbiBFKGMpe3RyeXt2YXIgYT10aGlzLnVwbG9hZCxiPWEuX19pZG1fcmVxaWRfXztpZihiKXt2YXIgZD1hLl9faWRtX2FjY2VwdF9fO2RlbGV0ZSBhLl9faWRtX2FjY2VwdF9fO2RlbGV0ZSBhLl9faWRtX3JlcWlkX187ZHx8ay5jYWxsKHRoaXMsImFjY2VwdCIsIiovKiIpO2suY2FsbCh0aGlzLCJhY2NlcHQiLAoiYXBwbGljYXRpb24vdm5kLnQxYy4iK2IpfX1jYXRjaChlKXt9cC5hcHBseSh0aGlzLGFyZ3VtZW50cyl9ZnVuY3Rpb24gQyhjKXt0cnl7aWYoYyl7dmFyIGE9dGhpcy5yZXNwb25zZVRleHQsYj12LmV4ZWMoYSk7cihbMTIyOTIxMjk3OSxjLGImJihiWzFdfHxhKXx8bnVsbF0sIi8iKX19Y2F0Y2goZCl7fX1mdW5jdGlvbiBGKGMsYSl7dmFyIGIsZCxlPXgoYyk7dHJ5e2lmKGI9dC50ZXN0KGU/Yy51cmw6Yykpe2U/ZD1jLmhlYWRlcnM6YT9kPWEuaGVhZGVyc3x8KGEuaGVhZGVycz1uZXcgSGVhZGVycyk6YT1hcmd1bWVudHNbMV09e2hlYWRlcnM6ZD1uZXcgSGVhZGVyc307dmFyIGw9ImZycS0iK3UrIi0iKyArK3c7ZC5oYXMoImFjY2VwdCIpfHxkLmFwcGVuZCgiYWNjZXB0IiwiKi8qIik7ZC5hcHBlbmQoImFjY2VwdCIsImFwcGxpY2F0aW9uL3ZuZC50MWMuIitsKX19Y2F0Y2goZil7fWQ9cS5hcHBseSh0aGlzLGFyZ3VtZW50cyk7dHJ5e2lmKGIpcmV0dXJuIGQudGhlbihHLmJpbmQodGhpcywKbCkpfWNhdGNoKGYpe31yZXR1cm4gZH1mdW5jdGlvbiBHKGMsYSl7dHJ5e2lmKGMpaWYoYS5vayl7dmFyIGI9YS50ZXh0LGQ9YS5qc29uO2EudGV4dD1mdW5jdGlvbigpe3JldHVybiBiLmNhbGwodGhpcykudGhlbih5LmJpbmQodGhpcyxjLCExKSl9O2EuanNvbj1mdW5jdGlvbigpe3JldHVybiBkLmNhbGwodGhpcykudGhlbih5LmJpbmQodGhpcyxjLCEwKSl9fWVsc2UgcihbMTIyOTIxMjk3OSxjXSwiLyIpfWNhdGNoKGUpe31yZXR1cm4gYX1mdW5jdGlvbiB5KGMsYSxiKXt0cnl7dmFyIGQ9YT9KU09OLnN0cmluZ2lmeShiKTpiLGU9di5leGVjKGQpO3IoWzEyMjkyMTI5NzksYyxlJiYoZVsxXXx8ZCl8fG51bGxdLCIvIil9Y2F0Y2gobCl7fXJldHVybiBifWZ1bmN0aW9uIHooYyxhKXt2YXIgYj1jO2E9YS5zcGxpdCgiLiIpO2lmKCIqIj09YVswXSl7YS5zaGlmdCgpO2Zvcih2YXIgZCBvZiBIKGMpKXRyeXtpZih4KGI9Y1tkXSkmJkkoYixhWzBdKSl7Zm9yKHZhciBlIG9mIGEpYj1iW2VdOwpyZXR1cm4gYn19Y2F0Y2gobCl7fX1lbHNlIHRyeXtmb3IoZSBvZiBhKWI9YltlXTtyZXR1cm4gYn1jYXRjaChsKXt9fWZ1bmN0aW9uIEEoYyxhKXtmb3IoO2MmJmEtLTtjPWMucGFyZW50RWxlbWVudCk7cmV0dXJuIGN9Y29uc3QgSj13aW5kb3cub3JpZ2lufHxkb2N1bWVudC5vcmlnaW58fGxvY2F0aW9uLm9yaWdpbix4PU9iamVjdC5pc0V4dGVuc2libGUsSz1BcnJheS5pc0FycmF5LGg9WE1MSHR0cFJlcXVlc3QucHJvdG90eXBlLHI9d2luZG93LnBvc3RNZXNzYWdlLmJpbmQod2luZG93KTt2YXIgdz0wLHU9MCx0LHYsbixwLGsscTtjb25zdCBJPU9iamVjdC5oYXNPd258fEZ1bmN0aW9uLmNhbGwuYmluZChPYmplY3QucHJvdG90eXBlLmhhc093blByb3BlcnR5KSxIPU9iamVjdC5nZXRPd25Qcm9wZXJ0eU5hbWVzLEw9LyAqXCQoWzw+PV0pICovO3dpbmRvdy5hZGRFdmVudExpc3RlbmVyKCJtZXNzYWdlIixmdW5jdGlvbihjKXt2YXIgYT1jLmRhdGE7aWYoSyhhKSYmYy5vcmlnaW49PQpKKXN3aXRjaChhWzBdKXtjYXNlIDEyMjkyMTI5Nzg6dT1hWzFdO3ZhciBiPWFbMl0sZD1hWzNdLGU9YVs0XTthPWFbNV07ZT8odD1SZWdFeHAoZSksdj1SZWdFeHAoYSkpOmI9ZD0hMTt0cnl7Yj8obnx8KG49aC5vcGVuLGgub3Blbj1CKSxwfHwocD1oLnNlbmQsaC5zZW5kPUUpLGt8fChrPWguc2V0UmVxdWVzdEhlYWRlcixoLnNldFJlcXVlc3RIZWFkZXI9RCkpOihuJiYoaC5vcGVuPW4sbj1udWxsKSxwJiYoaC5zZW5kPXAscD1udWxsKSxrJiYoaC5zZXRSZXF1ZXN0SGVhZGVyPWssaz1udWxsKSksZD9xfHwocT1mZXRjaCxmZXRjaD1GKTpxJiYoZmV0Y2g9cSxxPW51bGwpfWNhdGNoKE0pe31icmVhaztjYXNlIDEyMjkyMTI5ODA6Yz1hWzFdO3ZhciBsPWFbMl07YT1hWzNdO3ZhciBmO3RyeXtpZihsP2Y9ZG9jdW1lbnQucXVlcnlTZWxlY3RvcignW19faWRtX2lkX189IicrbCsnIl0nKTpmPXdpbmRvdyxmKWE6aWYoZD1hLnNwbGl0KEwpLDE8ZC5sZW5ndGgpe2ZvcihkLnNoaWZ0KCk7ZiYmCihlPWQuc2hpZnQoKSk7KXt2YXIgZz1kLnNoaWZ0KCkudHJpbSgpLG07Zm9yKG09MTsiPCI9PWdbZy5sZW5ndGgtbV07bSsrKTstLW0mJihnPWcuc2xpY2UoMCwtbSkudHJpbSgpKTtzd2l0Y2goZSl7Y2FzZSAiPCI6Zj1BKGc/Zi5jbG9zZXN0KGcpOmYsbSk7YnJlYWs7Y2FzZSAiPiI6Zj1BKGYucXVlcnlTZWxlY3RvcihnKSxtKTticmVhaztjYXNlICI9IjppZihnPXooZixnKSx2b2lkIDAhPT1nKXtiPWc7YnJlYWsgYX19fWI9dm9pZCAwfWVsc2UgYj16KGYsYSl9Y2F0Y2goTSl7fXIoWzEyMjkyMTI5ODEsYyxiXSwiLyIpfX0sITEpO3IoWzEyMjkyMTI5NzddLCIvIil9KSgpOwo=:2:40)

14:02:21 [ERROR] Sidebar unlock failed; Sidebar Unlock Failed
    at unlockResponse (<anonymous>:985:19)

14:02:21 [INFO] playerResponse property set, contains sidebar: true
14:02:21 [ERROR] Sidebar unlock failed; Sidebar Unlock Failed
    at unlockResponse (<anonymous>:985:19)

14:02:21 [ERROR] Video unlock failed; Player Unlock Failed, Proxy Error Message: Proxy Connection failed
    at unlockResponse$1 (<anonymous>:873:19)

14:02:21 [INFO] playerResponse property set, contains sidebar: false
14:02:21 [ERROR] Video unlock failed; Player Unlock Failed, Proxy Error Message: Proxy Connection failed
    at unlockResponse$1 (<anonymous>:873:19)

14:02:47 [INFO] Script injected (https://www.youtube.com/watch?v=J2XuYSx5jOo&ab_channel=melindalayla1369)
14:02:47 [INFO] playerResponse property set, contains sidebar: false
14:02:47 [INFO] Trying Player Unlock Method #2 (TV Embedded Player)
14:02:47 [INFO] Trying Player Unlock Method #4 (Account Proxy)
14:02:47 [ERROR] Proxy API Error; Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'https://youtube-proxy.zerody.one/getPlayer?videoId=J2XuYSx5jOo&reason=LOGIN_REQUIRED&clientName=WEB&clientVersion=2.20241025.01.00&signatureTimestamp=20019&startTimeSecs=0&hl=pl&isEmbed=0&isConfirmed=0&includeNext=1&client=js'.
    at self.XMLHttpRequest.E [as send] (data:text/javascript;base64,KGZ1bmN0aW9uKCl7ZnVuY3Rpb24gQihjLGEpe24uYXBwbHkodGhpcyxhcmd1bWVudHMpO3RyeXtpZih0LnRlc3QoYSkpe3ZhciBiPXRoaXMudXBsb2FkLl9faWRtX3JlcWlkX189InhycS0iK3UrIi0iKyArK3c7dGhpcy5hZGRFdmVudExpc3RlbmVyKCJsb2FkZW5kIixDLmJpbmQodGhpcyxiKSl9fWNhdGNoKGQpe319ZnVuY3Rpb24gRChjLGEpe3RyeXt0aGlzLnVwbG9hZC5fX2lkbV9yZXFpZF9fJiYiYWNjZXB0Ij09Yy50b0xvd2VyQ2FzZSgpJiYodGhpcy51cGxvYWQuX19pZG1fYWNjZXB0X189ITApfWNhdGNoKGIpe31rLmFwcGx5KHRoaXMsYXJndW1lbnRzKX1mdW5jdGlvbiBFKGMpe3RyeXt2YXIgYT10aGlzLnVwbG9hZCxiPWEuX19pZG1fcmVxaWRfXztpZihiKXt2YXIgZD1hLl9faWRtX2FjY2VwdF9fO2RlbGV0ZSBhLl9faWRtX2FjY2VwdF9fO2RlbGV0ZSBhLl9faWRtX3JlcWlkX187ZHx8ay5jYWxsKHRoaXMsImFjY2VwdCIsIiovKiIpO2suY2FsbCh0aGlzLCJhY2NlcHQiLAoiYXBwbGljYXRpb24vdm5kLnQxYy4iK2IpfX1jYXRjaChlKXt9cC5hcHBseSh0aGlzLGFyZ3VtZW50cyl9ZnVuY3Rpb24gQyhjKXt0cnl7aWYoYyl7dmFyIGE9dGhpcy5yZXNwb25zZVRleHQsYj12LmV4ZWMoYSk7cihbMTIyOTIxMjk3OSxjLGImJihiWzFdfHxhKXx8bnVsbF0sIi8iKX19Y2F0Y2goZCl7fX1mdW5jdGlvbiBGKGMsYSl7dmFyIGIsZCxlPXgoYyk7dHJ5e2lmKGI9dC50ZXN0KGU/Yy51cmw6Yykpe2U/ZD1jLmhlYWRlcnM6YT9kPWEuaGVhZGVyc3x8KGEuaGVhZGVycz1uZXcgSGVhZGVycyk6YT1hcmd1bWVudHNbMV09e2hlYWRlcnM6ZD1uZXcgSGVhZGVyc307dmFyIGw9ImZycS0iK3UrIi0iKyArK3c7ZC5oYXMoImFjY2VwdCIpfHxkLmFwcGVuZCgiYWNjZXB0IiwiKi8qIik7ZC5hcHBlbmQoImFjY2VwdCIsImFwcGxpY2F0aW9uL3ZuZC50MWMuIitsKX19Y2F0Y2goZil7fWQ9cS5hcHBseSh0aGlzLGFyZ3VtZW50cyk7dHJ5e2lmKGIpcmV0dXJuIGQudGhlbihHLmJpbmQodGhpcywKbCkpfWNhdGNoKGYpe31yZXR1cm4gZH1mdW5jdGlvbiBHKGMsYSl7dHJ5e2lmKGMpaWYoYS5vayl7dmFyIGI9YS50ZXh0LGQ9YS5qc29uO2EudGV4dD1mdW5jdGlvbigpe3JldHVybiBiLmNhbGwodGhpcykudGhlbih5LmJpbmQodGhpcyxjLCExKSl9O2EuanNvbj1mdW5jdGlvbigpe3JldHVybiBkLmNhbGwodGhpcykudGhlbih5LmJpbmQodGhpcyxjLCEwKSl9fWVsc2UgcihbMTIyOTIxMjk3OSxjXSwiLyIpfWNhdGNoKGUpe31yZXR1cm4gYX1mdW5jdGlvbiB5KGMsYSxiKXt0cnl7dmFyIGQ9YT9KU09OLnN0cmluZ2lmeShiKTpiLGU9di5leGVjKGQpO3IoWzEyMjkyMTI5NzksYyxlJiYoZVsxXXx8ZCl8fG51bGxdLCIvIil9Y2F0Y2gobCl7fXJldHVybiBifWZ1bmN0aW9uIHooYyxhKXt2YXIgYj1jO2E9YS5zcGxpdCgiLiIpO2lmKCIqIj09YVswXSl7YS5zaGlmdCgpO2Zvcih2YXIgZCBvZiBIKGMpKXRyeXtpZih4KGI9Y1tkXSkmJkkoYixhWzBdKSl7Zm9yKHZhciBlIG9mIGEpYj1iW2VdOwpyZXR1cm4gYn19Y2F0Y2gobCl7fX1lbHNlIHRyeXtmb3IoZSBvZiBhKWI9YltlXTtyZXR1cm4gYn1jYXRjaChsKXt9fWZ1bmN0aW9uIEEoYyxhKXtmb3IoO2MmJmEtLTtjPWMucGFyZW50RWxlbWVudCk7cmV0dXJuIGN9Y29uc3QgSj13aW5kb3cub3JpZ2lufHxkb2N1bWVudC5vcmlnaW58fGxvY2F0aW9uLm9yaWdpbix4PU9iamVjdC5pc0V4dGVuc2libGUsSz1BcnJheS5pc0FycmF5LGg9WE1MSHR0cFJlcXVlc3QucHJvdG90eXBlLHI9d2luZG93LnBvc3RNZXNzYWdlLmJpbmQod2luZG93KTt2YXIgdz0wLHU9MCx0LHYsbixwLGsscTtjb25zdCBJPU9iamVjdC5oYXNPd258fEZ1bmN0aW9uLmNhbGwuYmluZChPYmplY3QucHJvdG90eXBlLmhhc093blByb3BlcnR5KSxIPU9iamVjdC5nZXRPd25Qcm9wZXJ0eU5hbWVzLEw9LyAqXCQoWzw+PV0pICovO3dpbmRvdy5hZGRFdmVudExpc3RlbmVyKCJtZXNzYWdlIixmdW5jdGlvbihjKXt2YXIgYT1jLmRhdGE7aWYoSyhhKSYmYy5vcmlnaW49PQpKKXN3aXRjaChhWzBdKXtjYXNlIDEyMjkyMTI5Nzg6dT1hWzFdO3ZhciBiPWFbMl0sZD1hWzNdLGU9YVs0XTthPWFbNV07ZT8odD1SZWdFeHAoZSksdj1SZWdFeHAoYSkpOmI9ZD0hMTt0cnl7Yj8obnx8KG49aC5vcGVuLGgub3Blbj1CKSxwfHwocD1oLnNlbmQsaC5zZW5kPUUpLGt8fChrPWguc2V0UmVxdWVzdEhlYWRlcixoLnNldFJlcXVlc3RIZWFkZXI9RCkpOihuJiYoaC5vcGVuPW4sbj1udWxsKSxwJiYoaC5zZW5kPXAscD1udWxsKSxrJiYoaC5zZXRSZXF1ZXN0SGVhZGVyPWssaz1udWxsKSksZD9xfHwocT1mZXRjaCxmZXRjaD1GKTpxJiYoZmV0Y2g9cSxxPW51bGwpfWNhdGNoKE0pe31icmVhaztjYXNlIDEyMjkyMTI5ODA6Yz1hWzFdO3ZhciBsPWFbMl07YT1hWzNdO3ZhciBmO3RyeXtpZihsP2Y9ZG9jdW1lbnQucXVlcnlTZWxlY3RvcignW19faWRtX2lkX189IicrbCsnIl0nKTpmPXdpbmRvdyxmKWE6aWYoZD1hLnNwbGl0KEwpLDE8ZC5sZW5ndGgpe2ZvcihkLnNoaWZ0KCk7ZiYmCihlPWQuc2hpZnQoKSk7KXt2YXIgZz1kLnNoaWZ0KCkudHJpbSgpLG07Zm9yKG09MTsiPCI9PWdbZy5sZW5ndGgtbV07bSsrKTstLW0mJihnPWcuc2xpY2UoMCwtbSkudHJpbSgpKTtzd2l0Y2goZSl7Y2FzZSAiPCI6Zj1BKGc/Zi5jbG9zZXN0KGcpOmYsbSk7YnJlYWs7Y2FzZSAiPiI6Zj1BKGYucXVlcnlTZWxlY3RvcihnKSxtKTticmVhaztjYXNlICI9IjppZihnPXooZixnKSx2b2lkIDAhPT1nKXtiPWc7YnJlYWsgYX19fWI9dm9pZCAwfWVsc2UgYj16KGYsYSl9Y2F0Y2goTSl7fXIoWzEyMjkyMTI5ODEsYyxiXSwiLyIpfX0sITEpO3IoWzEyMjkyMTI5NzddLCIvIil9KSgpOwo=:2:40)

14:02:47 [ERROR] Video unlock failed; Player Unlock Failed, Proxy Error Message: Proxy Connection failed
    at unlockResponse$1 (<anonymous>:873:19)

14:02:48 [INFO] Trying Next Unlock Method #2 (Account Proxy)
14:02:48 [ERROR] Proxy API Error; Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'https://youtube-proxy.zerody.one/getNext?videoId=J2XuYSx5jOo&clientName=WEB&clientVersion=2.20241025.01.00&hl=pl&userInterfaceTheme=USER_INTERFACE_THEME_DARK&isEmbed=0&isConfirmed=0&client=js'.
    at self.XMLHttpRequest.E [as send] (data:text/javascript;base64,KGZ1bmN0aW9uKCl7ZnVuY3Rpb24gQihjLGEpe24uYXBwbHkodGhpcyxhcmd1bWVudHMpO3RyeXtpZih0LnRlc3QoYSkpe3ZhciBiPXRoaXMudXBsb2FkLl9faWRtX3JlcWlkX189InhycS0iK3UrIi0iKyArK3c7dGhpcy5hZGRFdmVudExpc3RlbmVyKCJsb2FkZW5kIixDLmJpbmQodGhpcyxiKSl9fWNhdGNoKGQpe319ZnVuY3Rpb24gRChjLGEpe3RyeXt0aGlzLnVwbG9hZC5fX2lkbV9yZXFpZF9fJiYiYWNjZXB0Ij09Yy50b0xvd2VyQ2FzZSgpJiYodGhpcy51cGxvYWQuX19pZG1fYWNjZXB0X189ITApfWNhdGNoKGIpe31rLmFwcGx5KHRoaXMsYXJndW1lbnRzKX1mdW5jdGlvbiBFKGMpe3RyeXt2YXIgYT10aGlzLnVwbG9hZCxiPWEuX19pZG1fcmVxaWRfXztpZihiKXt2YXIgZD1hLl9faWRtX2FjY2VwdF9fO2RlbGV0ZSBhLl9faWRtX2FjY2VwdF9fO2RlbGV0ZSBhLl9faWRtX3JlcWlkX187ZHx8ay5jYWxsKHRoaXMsImFjY2VwdCIsIiovKiIpO2suY2FsbCh0aGlzLCJhY2NlcHQiLAoiYXBwbGljYXRpb24vdm5kLnQxYy4iK2IpfX1jYXRjaChlKXt9cC5hcHBseSh0aGlzLGFyZ3VtZW50cyl9ZnVuY3Rpb24gQyhjKXt0cnl7aWYoYyl7dmFyIGE9dGhpcy5yZXNwb25zZVRleHQsYj12LmV4ZWMoYSk7cihbMTIyOTIxMjk3OSxjLGImJihiWzFdfHxhKXx8bnVsbF0sIi8iKX19Y2F0Y2goZCl7fX1mdW5jdGlvbiBGKGMsYSl7dmFyIGIsZCxlPXgoYyk7dHJ5e2lmKGI9dC50ZXN0KGU/Yy51cmw6Yykpe2U/ZD1jLmhlYWRlcnM6YT9kPWEuaGVhZGVyc3x8KGEuaGVhZGVycz1uZXcgSGVhZGVycyk6YT1hcmd1bWVudHNbMV09e2hlYWRlcnM6ZD1uZXcgSGVhZGVyc307dmFyIGw9ImZycS0iK3UrIi0iKyArK3c7ZC5oYXMoImFjY2VwdCIpfHxkLmFwcGVuZCgiYWNjZXB0IiwiKi8qIik7ZC5hcHBlbmQoImFjY2VwdCIsImFwcGxpY2F0aW9uL3ZuZC50MWMuIitsKX19Y2F0Y2goZil7fWQ9cS5hcHBseSh0aGlzLGFyZ3VtZW50cyk7dHJ5e2lmKGIpcmV0dXJuIGQudGhlbihHLmJpbmQodGhpcywKbCkpfWNhdGNoKGYpe31yZXR1cm4gZH1mdW5jdGlvbiBHKGMsYSl7dHJ5e2lmKGMpaWYoYS5vayl7dmFyIGI9YS50ZXh0LGQ9YS5qc29uO2EudGV4dD1mdW5jdGlvbigpe3JldHVybiBiLmNhbGwodGhpcykudGhlbih5LmJpbmQodGhpcyxjLCExKSl9O2EuanNvbj1mdW5jdGlvbigpe3JldHVybiBkLmNhbGwodGhpcykudGhlbih5LmJpbmQodGhpcyxjLCEwKSl9fWVsc2UgcihbMTIyOTIxMjk3OSxjXSwiLyIpfWNhdGNoKGUpe31yZXR1cm4gYX1mdW5jdGlvbiB5KGMsYSxiKXt0cnl7dmFyIGQ9YT9KU09OLnN0cmluZ2lmeShiKTpiLGU9di5leGVjKGQpO3IoWzEyMjkyMTI5NzksYyxlJiYoZVsxXXx8ZCl8fG51bGxdLCIvIil9Y2F0Y2gobCl7fXJldHVybiBifWZ1bmN0aW9uIHooYyxhKXt2YXIgYj1jO2E9YS5zcGxpdCgiLiIpO2lmKCIqIj09YVswXSl7YS5zaGlmdCgpO2Zvcih2YXIgZCBvZiBIKGMpKXRyeXtpZih4KGI9Y1tkXSkmJkkoYixhWzBdKSl7Zm9yKHZhciBlIG9mIGEpYj1iW2VdOwpyZXR1cm4gYn19Y2F0Y2gobCl7fX1lbHNlIHRyeXtmb3IoZSBvZiBhKWI9YltlXTtyZXR1cm4gYn1jYXRjaChsKXt9fWZ1bmN0aW9uIEEoYyxhKXtmb3IoO2MmJmEtLTtjPWMucGFyZW50RWxlbWVudCk7cmV0dXJuIGN9Y29uc3QgSj13aW5kb3cub3JpZ2lufHxkb2N1bWVudC5vcmlnaW58fGxvY2F0aW9uLm9yaWdpbix4PU9iamVjdC5pc0V4dGVuc2libGUsSz1BcnJheS5pc0FycmF5LGg9WE1MSHR0cFJlcXVlc3QucHJvdG90eXBlLHI9d2luZG93LnBvc3RNZXNzYWdlLmJpbmQod2luZG93KTt2YXIgdz0wLHU9MCx0LHYsbixwLGsscTtjb25zdCBJPU9iamVjdC5oYXNPd258fEZ1bmN0aW9uLmNhbGwuYmluZChPYmplY3QucHJvdG90eXBlLmhhc093blByb3BlcnR5KSxIPU9iamVjdC5nZXRPd25Qcm9wZXJ0eU5hbWVzLEw9LyAqXCQoWzw+PV0pICovO3dpbmRvdy5hZGRFdmVudExpc3RlbmVyKCJtZXNzYWdlIixmdW5jdGlvbihjKXt2YXIgYT1jLmRhdGE7aWYoSyhhKSYmYy5vcmlnaW49PQpKKXN3aXRjaChhWzBdKXtjYXNlIDEyMjkyMTI5Nzg6dT1hWzFdO3ZhciBiPWFbMl0sZD1hWzNdLGU9YVs0XTthPWFbNV07ZT8odD1SZWdFeHAoZSksdj1SZWdFeHAoYSkpOmI9ZD0hMTt0cnl7Yj8obnx8KG49aC5vcGVuLGgub3Blbj1CKSxwfHwocD1oLnNlbmQsaC5zZW5kPUUpLGt8fChrPWguc2V0UmVxdWVzdEhlYWRlcixoLnNldFJlcXVlc3RIZWFkZXI9RCkpOihuJiYoaC5vcGVuPW4sbj1udWxsKSxwJiYoaC5zZW5kPXAscD1udWxsKSxrJiYoaC5zZXRSZXF1ZXN0SGVhZGVyPWssaz1udWxsKSksZD9xfHwocT1mZXRjaCxmZXRjaD1GKTpxJiYoZmV0Y2g9cSxxPW51bGwpfWNhdGNoKE0pe31icmVhaztjYXNlIDEyMjkyMTI5ODA6Yz1hWzFdO3ZhciBsPWFbMl07YT1hWzNdO3ZhciBmO3RyeXtpZihsP2Y9ZG9jdW1lbnQucXVlcnlTZWxlY3RvcignW19faWRtX2lkX189IicrbCsnIl0nKTpmPXdpbmRvdyxmKWE6aWYoZD1hLnNwbGl0KEwpLDE8ZC5sZW5ndGgpe2ZvcihkLnNoaWZ0KCk7ZiYmCihlPWQuc2hpZnQoKSk7KXt2YXIgZz1kLnNoaWZ0KCkudHJpbSgpLG07Zm9yKG09MTsiPCI9PWdbZy5sZW5ndGgtbV07bSsrKTstLW0mJihnPWcuc2xpY2UoMCwtbSkudHJpbSgpKTtzd2l0Y2goZSl7Y2FzZSAiPCI6Zj1BKGc/Zi5jbG9zZXN0KGcpOmYsbSk7YnJlYWs7Y2FzZSAiPiI6Zj1BKGYucXVlcnlTZWxlY3RvcihnKSxtKTticmVhaztjYXNlICI9IjppZihnPXooZixnKSx2b2lkIDAhPT1nKXtiPWc7YnJlYWsgYX19fWI9dm9pZCAwfWVsc2UgYj16KGYsYSl9Y2F0Y2goTSl7fXIoWzEyMjkyMTI5ODEsYyxiXSwiLyIpfX0sITEpO3IoWzEyMjkyMTI5NzddLCIvIil9KSgpOwo=:2:40)

14:02:48 [ERROR] Sidebar unlock failed; Sidebar Unlock Failed
    at unlockResponse (<anonymous>:985:19)

14:02:48 [INFO] playerResponse property set, contains sidebar: true
14:02:48 [ERROR] Sidebar unlock failed; Sidebar Unlock Failed
    at unlockResponse (<anonymous>:985:19)

14:02:48 [ERROR] Video unlock failed; Player Unlock Failed, Proxy Error Message: Proxy Connection failed
    at unlockResponse$1 (<anonymous>:873:19)

14:02:48 [INFO] playerResponse property set, contains sidebar: false
14:02:48 [ERROR] Video unlock failed; Player Unlock Failed, Proxy Error Message: Proxy Connection failed
    at unlockResponse$1 (<anonymous>:873:19)

### Platform

Extension

### Version

newest

### What browser are you on?

Opera

### Browser version

newest

### Link to the video

https://www.youtube.com/watch?v=J2XuYSx5jOo

### Screenshots or Videos

![image](https://github.com/user-attachments/assets/6bd71b3f-c87f-446c-890c-c3b441710d5d)

### About you

- [ ] I am a developer and/or have knowledge in programming

**Comment by drunkwinter** [comment]
Duplicate of #262, #263.
